### DISCUSSION TITLE
UDMP beta firmware 1.11.0 supported?

### DISCUSSION THREAD

**Discussion by TheJulianJES** [fabianishere/udm-kernel-tools#23] [General]
If anyone tried it already, is UDM(P) (beta) firmware 1.11.0-x already supported? (Does the existing kernel "just work"?)
https://community.ui.com/releases/UniFi-OS-Dream-Machines-1-11-0-14/71916646-d8f6-41c0-b145-2fbe2db7c278

Kernel configuration: https://paste.ubuntu.com/p/QPYkFF2gFd/

This is new:
```
CONFIG_IP_MROUTE_COMMON=y
CONFIG_IP_MROUTE=y
# CONFIG_IP_MROUTE_MULTIPLE_TABLES is not set
# CONFIG_IP_PIMSM_V1 is not set
# CONFIG_IP_PIMSM_V2 is not set
```

Would these changes alone make it possible to run ``igmpproxy`` without a custom kernel maybe?

**Comment by fabianishere** [comment]
Yes, that is enough to run igmpproxy!

**Reply by roscog** [reply]
is this suported yet. and if i understand corectly this can be used for iptv?

**Reply by fabianishere** [reply]
With the beta firmware, you can just follow the [guide](https://github.com/fabianishere/udm-kernel-tools/blob/master/docs/iptv.md) to set up IPTV straight away, without needing to set up the custom kernel.

**Reply by roscog** [reply]
Is there a way to upload custom firmware to the udm,for me it only shows fully released firmware

**Reply by fabianishere** [reply]
You need to enroll for the beta program or manually install the firmware with the link from the release page. That page also contains instructions for installing it on your device.
